### ISSUE TITLE
[Bug]: Saved output file doesn't open anymore - internal crash

### ISSUE THREAD

**Issue by ukiana** [jasp-stats/jasp-issues#2459]
### JASP Version

0.18.1

### JASP Module

Unrelated

### What OS are you seeing the problem on?

macOS Silicon

### Bug Description

Hi,

I am having issues opening a file I worked on before in JASP. please find the file enclosed.

I have a Macbook Pro, macOS, version Sonoma 14.1.2.
![Screenshot 2023-12-05 at 20 39 21](https://github.com/jasp-stats/jasp-issues/assets/153021202/3ee32cbe-b41e-4999-9ae9-ef5225b8ba2c)

For JASP, I use the latest version 0.18.1.0-macOS-arm64, which worked totally fine until yesterday evening. Please also find below the last few logs of the app.

I need this file as I worked for University. So I'd appreciate to either guide me through how to solve the issue, or provide me with a copy so I can upload it.

For your reference, I tried to open the file on the 16 version of JASP. I also uninstalled and reinstalled JASP, but it still doesn't work.

Thank you,
Ana
[TeleMarket - 1st iteration.jasp.zip](https://github.com/jasp-stats/jasp-issues/files/13578598/TeleMarket.-.1st.iteration.jasp.zip)

### Expected Behaviour

Open the file in JASP desktop app, access the input and output data 

### Steps to Reproduce

1. Open the file without crash and access the data

### Log (if any)

[JASP logs.zip](https://github.com/jasp-stats/jasp-issues/files/13578635/JASP.logs.zip)

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by boutinb** [comment]
Hi @ukiana Unfortunately your JASP file is quite corrupted: the column names have no name, and I cannot open it. If I have a look at its version, I see it was built with a 0.14. Can you still open it with this version?

**Comment by ukiana** [comment]
Hi @boutinb thank you for looking into this.

I tried older versions, and this is the error I get. I tried both 0.14 and 0.14.1.
![Screenshot 2023-12-06 at 15 43 02](https://github.com/jasp-stats/jasp-issues/assets/153021202/46ef1b37-0fe8-41b7-a42a-77113f547194)

What could have caused this issue? All was alright until last night when I tried to open it for the last time.
Is there a way to convert this file into a csv or some other format that could be uploaded intp JASP?
